src/librustdoc/html/static/rustdoc.css

```diff
@@ -1569,9 +1569,13 @@ h4 > .notable-traits {
 		height: 73px;
 	}
 
+	#main {
+		margin-top: 100px;
```

Comment:
This results in quite a big blank area on desktop

![image](https://user-images.githubusercontent.com/81079/102087210-80be6d00-3e19-11eb-8a09-715011585cf0.png)

Reply:
I was about to comment and ask how you were able to achieve that result. :rofl:

Reply:
Manually editing the css on doc.rust-lang.org wrong to test it :sweat_smile:

Reply:
Just in this case, generally it works. ;)